### ISSUE TITLE
UDL Styler Dialog crash

### ISSUE THREAD

**Issue by FatalBulletHit** [notepad-plus-plus/notepad-plus-plus#4275]
### Description of the Issue
When hitting enter after typing either the name or the size of a font in the Styler Dialog of the User Defined Language, Notepad++ crashes.

### Steps to Reproduce the Issue
1.  Language -> Define your language...
2.  Styler
3.  Type something in Name or Size and hit enter

### Expected Behavior
Apply changes made to name or size of the font

### Actual Behavior
Notepad++ crashes

**Comment by SinghRajenM** [comment]
No luck. Please paste Debug info here.

**Comment by FatalBulletHit** [comment]
Don't know how to get the debug info.
But for now I've a .gif for you:

![grafik](https://user-images.githubusercontent.com/37146617/37106012-f42bd578-2230-11e8-8226-86dd3072e753.png)

Hitting enter or escape will trigger this when the Name or Size field is still active.

**Comment by sasumner** [comment]
@FatalBulletHit Get debug info on the ? menu
? -> Debug Info...

Copy it and paste it here

**Comment by FatalBulletHit** [comment]
There you go:

```
Notepad++ v7.5.4   (64-bit)
Build time : Jan  1 2018 - 01:50:29
Path : C:\Program Files\Notepad++\notepad++.exe
Admin mode : OFF
Local Conf mode : OFF
OS : Windows 10 (64-bit)
Plugins : DSpellCheck.dll mimeTools.dll NppConverter.dll 

```

**Comment by SinghRajenM** [comment]
Got it. Duplicate of #2646 and fixed in #4279

**Comment by FatalBulletHit** [comment]
Thanks! :)
